Source/ACE.DatLoader/ACE.DatLoader.csproj

```diff
@@ -90,11 +109,16 @@
       <Project>{136e260e-b4a8-4e6f-b9cb-6ae7fc368dc8}</Project>
       <Name>ACE.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\ACE.Entity\ACE.Entity.csproj">
+      <Project>{aa611a84-cdf8-4ab6-8d53-085c897a7422}</Project>
+      <Name>ACE.Entity</Name>
+    </ProjectReference>
   </ItemGroup>
   <ItemGroup>
     <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup />

```

Comment:
Not sure what/how that got there. Something Visual Studio did for me, apparently. I'll remove it just to keep things clean, though.